src/main/java/mate/academy/spring/config/SecurityConfig.java

```diff
@@ -22,6 +23,22 @@ public void configureGlobal(AuthenticationManagerBuilder auth) throws Exception
     protected void configure(HttpSecurity http) throws Exception {
```

Comment:
let's also configure DB authentication instead of in-memory using your custom UserDetailsService